$OpenBSD: patch-ed25519_cabal,v 1.1.1.1 2017/11/03 14:31:22 kili Exp $

Dependency changed in the "online" version of the package description,
but not in the dist file.

--- ed25519.cabal.orig	Mon Oct 12 01:24:33 2015
+++ ed25519.cabal	Thu Dec 29 20:06:27 2016
@@ -60,7 +60,7 @@ flag no-donna
 
 library
   build-depends:
-    ghc-prim    >= 0.1 && < 0.5,
+    ghc-prim    >= 0.1 && < 0.6,
     base        >= 4   && < 5,
     bytestring  >= 0.9 && < 0.11
 
